scene/resources/theme.cpp

```diff
@@ -1250,14 +1250,12 @@ void Theme::get_type_list(List<StringName> *p_list) const {
 	}
 }
 
-void Theme::get_type_dependencies(const StringName &p_base_type, const StringName &p_type_variation, List<StringName> *p_list) {
-	ERR_FAIL_NULL(p_list);
```

Comment:
Is this basically guaranteed to never happen, hence the removal of this line?

Reply:
It's not passed by pointer any more but by reference